src/expandQuickLinks.js

```diff
@@ -0,0 +1,8 @@
+let navMain = document.getElementById('nav-main');
+let quickLinks = document.getElementById('quick-links');
+
+navMain.appendChild(quickLinks.getElementsByClassName('icon-search-self')[0]);
+navMain.appendChild(quickLinks.getElementsByClassName('icon-search-new')[0]);
```

Comment:
Will fail if user isn't signed in;, perhaps use absolute URLs for all.

Reply:
correct
I will fix it once the settings page is done
and I have feedback on the second style I have posted

Thanks for your review